groth16-framework/test_data/Groth16VerifierExtensions.sol

```diff
@@ -208,6 +248,26 @@ contract Query2 is Verifier {
         return result;
     }
 
+    // Convert a bytes32 to an uint256.
+    function convertBytes32ToU256(bytes32 b) internal pure returns (uint256) {
```

Comment:
Why can't you just do `bytes32(uint256)` ?

Reply:
Seems the endian is not right, this function is just renamed from [convertToU256](https://github.com/Lagrange-Labs/lagrange-lpn-contracts/blob/master/src/Groth16VerifierExtensions.sol#L210C14-L210C27) before.